splinepy/io/gismo.py

```diff
@@ -45,6 +48,22 @@ def _spline_to_ET(
     if fields_only and (len(multipatch.fields) == 0):
         return
 
+    if fields_only and field_mask is not None:
+        if not isinstance(field_mask, (list, _np.ndarray)):
+            raise ValueError("field_mask must be list of integers")
+        field_mask = _np.unique(
+            _enforce_contiguous(field_mask, dtype=_np.int64, asarray=True)
+        )
+
+        # Check if range is valid
+        if (field_mask.min() < 0) or (
+            field_mask.max() >= len(multipatch.fields)
```

Comment:
```suggestion
        if (min(field_mask) < 0) or (
            max(field_mask) >= len(multipatch.fields)
```
this should be applicable for any array-like obj

Reply:
is always numpy

Reply:
ok